api-server/src/uploads/uploads.service.ts

```diff
@@ -11,6 +11,10 @@ import { PATH_TO_LOCAL_EVENT_IMAGE_UPLOADS } from './uploads.module';
 import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
 import { isNullOrUndefined } from '../utils';
 import { S3 } from '@aws-sdk/client-s3';
+import sharp, { Sharp } from 'sharp';
+
+const IMAGE_EXTENSIONS = 'webp';
```

Comment:
Nitpick but since this is being used later to set the extension, shouldn't it be `IMAGE_EXTENSION`?

Reply:
yep, fixed bd3116e334f3cc2b26975ecb6eb32a8b6ad4cc62